examples/seismic/tti/tti_example.py

```diff
@@ -62,22 +56,24 @@ def run(dimensions=(50, 50, 50), spacing=(20.0, 20.0, 20.0), tn=250.0,
                         help="Enable autotuning for block sizes")
     parser.add_argument("-to", "--time_order", default=2,
                         type=int, help="Time order of the simulation")
-    parser.add_argument("-so", "--space_order", default=4,
+    parser.add_argument("-so", "--space_order", default=8,
                         type=int, help="Space order of the simulation")
-    parser.add_argument("--nbpml", default=10,
+    parser.add_argument("--nbpml", default=40,
                         type=int, help="Number of PML layers around the domain")
     args = parser.parse_args()
 
     # 3D preset parameters
     if args.dim2:
         dimensions = (150, 150)
-        spacing = (15.0, 15.0)
-        tn = 750.0
+        spacing = (10.0, 10.0)
+        tn = 500.0
+        dse = 'basic'
     else:
-        dimensions = (50, 50, 50)
-        spacing = (20.0, 20.0, 20.0)
-        tn = 250.0
+        dimensions = (100, 100, 100)
+        spacing = (10.0, 10.0, 10.0)
+        tn = 400.0
+        dse = 'aggressive'
```

Comment:
Can you not just change the default values in the function signature ? 
I wouldn't set ``dse`` here; you can add it as a parser option if you really want.

Also, can we not fetch some of these params from the model ?

Reply:
only shows the full diff from size changes, dse here due to current 2d/3d issues with dse, will go away once figured out

Reply:
>Can we not fetch some of these params from the model ?
Well no, this params create the model
>Can you not just change the default values in the function signature ?
Nothing changed from before, only the dse due to the current 2d/3d behavior bugs